rllib/offline/estimators/qreg_torch_model.py

```diff
@@ -40,18 +41,25 @@ def __init__(
                         "fcnet_activation": "relu",
                         "vf_share_layers": True,
                     },
-            # Maximum number of training iterations to run on the batch
-            n_iters = 160,
+            # Number of training iterations to run on the batch
+            n_iters = 1,
             # Learning rate for Q-function optimizer
             lr = 1e-3,
             # Early stopping if the mean loss < delta
             delta = 1e-4,
             # Clip gradients to this maximum value
             clip_grad_norm = 100.0,
-            # Minibatch size for training Q-function
-            batch_size = 32,
+            # Minibatch size for training Q-function;
+            # if None, train on the whole batch
+            minibatch_size = None,
         """
         self.policy = policy
+        assert isinstance(
+            policy.action_space, Discrete
+        ), f"{self.__class__.__name__} only supports discrete action spaces!"
+        assert (
+            policy.config["batch_mode"] == "complete_episodes"
```

Comment:
**todo**: policy.config["evaluation"]["batch_mode"]

Reply:
Resolved by dropping Q-Reg; @kouroshHakha can you remove the change request? It blocks merge